modules/tinymce/src/models/dom/test/ts/browser/table/command/TableDeleteColumnTest.ts

```diff
@@ -117,7 +117,7 @@ describe('browser.tinymce.models.dom.table.command.TableDeleteColumnTest', () =>
       const lastColIndex = cols - 1;
       const pathToBody = [ 0, options.colgroup ? 1 : 0 ];
 
-      before(() => {
+      beforeEach(() => {
```

Comment:
`before` caused this test to be very temperemental due to relying on the test before it. Changed it to beforeEach.
This actually has nothing to do with my changes, but it kept failing on me ( due to the new ordering from my one added test? ) so I fixed it up a bit.